### DISCUSSION TITLE
Where do the tagging keys come from on a poi?

### DISCUSSION THREAD

**Discussion by mcliquid** [Zverik/every_door#497] [Q&A] [answered]
Where do the possible keys for POI's come from? Is there a list or how can I extend it?

**Comment by mnalis** [comment, marked as answer]
You mean [iD tagging schema](https://github.com/openstreetmap/id-tagging-schema), like in this issue https://github.com/Zverik/every_door/issues/449 ?

Also see `About` / `FAQ` / `All tagging questions` in the app for instructions on https://github.com/Zverik/every_door/blob/main/lib/helpers/good_tags.dart
